src/gsy_e/models/strategy/energy_parameters/virtual_heat_pump.py

```diff
@@ -0,0 +1,196 @@
+import logging
+from typing import Optional, Union, Dict
+
+import sympy as sp
+from gsy_framework.constants_limits import ConstSettings, GlobalConfig
+from gsy_framework.read_user_profile import InputProfileTypes
+from gsy_framework.utils import convert_W_to_kWh, convert_kWh_to_W
+from pendulum import DateTime
+
+from gsy_e.constants import FLOATING_POINT_TOLERANCE
+from gsy_e.models.strategy.energy_parameters.heat_pump import (
+    HeatPumpEnergyParametersBase, WATER_DENSITY)
+from gsy_e.models.strategy.profile import EnergyProfile
+
+logger = logging.getLogger(__name__)
+
+CALIBRATION_COEFFICIENT = 0.85
+WATER_SPECIFIC_TEMPERATURE = 4182
```

Comment:
```suggestion
WATER_SPECIFIC_HEAT_CAPACITY = 4182  # [J/kg°C]
```

Reply:
Again correcting the experts, WATER_SPECIFIC_TEMPERATURE was the term used on Mustafa's document but you are obviously correct.